doc/nrf-bm/migration/nrf5_bm_migration.rst

```diff
@@ -3,298 +3,6 @@
 Migration notes - nRF5 SDK to nRF Connect SDK Bare Metal
 ########################################################
 
-This document outlines the high-level differences between nRF5 SDK and the |BMlong|.
+This document is maintained in the latest version of |BMshort| documentation:
```

Comment:
I think we should keep what we have. If we see the need to point to the latest as the migration guide is less mature, we could have a note in the top in addition.

The reason is that when we move forward with the project and make changes to the codebase, the latest migration guide might not match what is in this version. Hence, in the future when we have say v1.2.0 and someone is working on v1.0.0, they will not have a migration guide that corresponds with what they have.

@bjorn-tore-taraldsen Any comments?

Reply:
I just wrote a 500 words comment why we should keep it as a link to latest but the more I look at the doc, the more problematic it seems 🤔  I'm just going to bring back the content and let's keep it in the release documentation after all.

Reply:
We discussed, in todays standup, what will be the best approach for the migration doc. 
Conclusion - We should keep this within the release (fixed) as Eivind suggest.